Makefile

```diff
@@ -44,7 +44,7 @@ run-dev: load-env
 	RUST_LOG=trace cargo run --bin kakarot-rpc
 
 install-katana:
-	cargo install --git https://github.com/dojoengine/dojo --locked --tag v1.0.0-alpha.14 katana
+	cargo install --git https://github.com/dojoengine/dojo --locked --tag v1.0.0-alpha.15 katana
```

Comment:
Can we have the same version also on cargo.toml and everywhere or is there a blocker?